tests/unit/test_search.py

```diff
@@ -501,6 +874,48 @@ def test_multiple_parameters(query_handler):
     assert len(response['matches']['propositions']) > 1
 
 
+def test_civic_detail_flag(query_handler, civic_eid2997, eid2997_proposition,
```

Comment:
It'd probably have a really ugly method signature, but would it be possible/helpful to write a helper function to perform the logic for these detail flag tests and have the CIViC and MOA test cases call it? Might make adding more sources later a bit easier

Reply:
Yup, I'll add it!

Reply:
Added in this commit https://github.com/cancervariants/metakb/pull/79/commits/45bff62b7513f4d67b010ae82caa2ec758661785